test/controllers/email/VerifyPasscodeControllerSpec.scala

```diff
@@ -244,10 +249,17 @@ class VerifyPasscodeControllerSpec extends ControllerBaseSpec with MockEmailVeri
 
       "there is an email in session" should {
 
-        lazy val result = TestVerifyPasscodeController.emailSendVerification()(requestWithEmail)
+        lazy val result = {
```

Comment:
Can you also add tests for the `emailSendVerification` action when calling the service function for the email passcode request. Currently untested scenarios are: email is already verified, unexpected error response.